pkg/route/api/types.go

```diff
@@ -25,9 +25,9 @@ type RouteSpec struct {
 	// Path that the router watches for, to route traffic for to the service. Optional
 	Path string
 
-	// An object the route points to. Only the Service kind is allowed, and it will
+	// Objects that the route points to. Only the Service kind is allowed, and it will
 	// be defaulted to Service.
-	To kapi.ObjectReference
+	To []kapi.ObjectReference
 
```

Comment:
Don't do this - keep the fields the same internally and externally.  When we move to ingress this is going to have to merge in, and ingress has no concept of duplicate from.